server/startup/index.js

```diff
@@ -12,7 +11,6 @@ export default function () {
   Accounts();
```

Comment:
That definitely occurred to me, but I opted not to because that meant I should probably move all of those files from `/server/startup` to `/server/api/core`.  There's certainly nothing wrong/difficult about doing that, but I was trying to keep things as simple as possible for the sake of a clear PR.  I'm totally not against it though.  I think it makes a lot of sense to kick everything off right inside the init function.  That makes it way easier to see what's going on and in what order.